recipes/multi-rake/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "multi_rake" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 885ab2fd795cb96015aa552f22b35b8ae9065ae2d2d5c1af42a1bb32e081bec4
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - cld2-cffi >=0.1.4
+    - numpy >=1.14.4
+    - pip
+    - pyrsistent >=0.14.2
+    - python >=3.5
+    - regex >=2018.6.6
```

Comment:
Do you really need most of these at install time?

Reply:
@scopatz thanks, host reqs have been updated